### ISSUE TITLE
Selection of displayed items in the menu

### ISSUE THREAD

**Issue by Mrakobes666** [gildas-lormeau/SingleFile#859]
For more than a year I have been using the extension every day, but only one of the items "save selected" in the nested drop-down menu - how can I move an item to the main menu and hide all the others?
![2022-01-09 12-31-14](https://user-images.githubusercontent.com/34971439/148742006-1b0493cc-cb89-4730-ab28-edb14a38c304.png)

**Comment by gildas-lormeau** [comment]
I understand your issue. Today, this feature does not exist. However, you could maybe use `Ctrl+Shift+Y` instead?

**Comment by Mrakobes666** [comment]
Yes, hotkeys work - it's still inconvenient, you can't remember all the combinations, and you have to take several steps to get to the goal
